### PR TITLE
Database file structure

### PR DESCRIPTION
**Pull request by g4ss4n**
#2 #3 #4 #5

### CODE REVIEW

**Comment by g4ss4n** [comment]
Make sure you install the NPM packages that I added

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const path = require("path");
+const express = require("express");
+require("env2")("config.env");
+
+const app = express();
+const getData = require("./database/models/GetData");
+
+
+var bodyParser = require("body-parser");
```
Comment (Karyum) [inline src/server.js]: ooof this `var` is a bit inconsistent

File: src/server.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const path = require("path");
+const express = require("express");
+require("env2")("config.env");
+
+const app = express();
+const getData = require("./database/models/GetData");
```
Comment (Karyum) [inline src/server.js]: hmm calling the file getData means you guys are not yet familiar with what is actually a model.
usually each database table has its own model, and that model controls data fetched from that table and inserted, relations between that table and etc. 
so i would rather have that file called `Tickets` :sweat_smile:

File: src/database/models/GetData.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      reject(err);
+    }else{
+      resolve(res.rows);
+    }
+  });
+  })
+  
+};
+
+const getArtists =()=> {
```
Comment (Karyum) [inline src/database/models/GetData.js]: oh and a seperate model for artists

**Comment by ayahkh** [comment]
the config.env file got pushed although I put it in the gitignore

File: config.env [hunk 4]
```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://ppbqwqswhxadys:6dfadf7d130a316bb15aadfda1174eac395b995f99a6f02cac9f5bc63dcd099f@ec2-107-20-167-241.compute-1.amazonaws.com:5432/dahnbjouovkoht
```
Comment (Karyum) [inline config.env]: ohh no 😄

**Comment by Karyum** [comment]
@ayahkh so you just need to delete it and then push to the branch.

**Pull request merged**